adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/V3.kt

```diff
@@ -47,6 +48,7 @@ problems as an issue in the <a href=\"https://github.com/AdoptOpenJDK/openjdk-ap
 )
 @ApplicationScoped
 @ApplicationPath("/")
+@Startup
```

Comment:
RESTEasy classic does eagerly initialize the jax-rs Application class, RESTEasy Reactive does not.
It's not a problem without @Startup in general, but a change in behavior, as without @Startup the service would run even without MongoDB connection.

Should be the only difference of that kind according to the answer from the Quarkus team: https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/Resteasy.20Reactive.3A.20Application.20class.20init